spec/shr_datatype_vs.txt

```diff
@@ -307,6 +256,21 @@ Description:		"A code indicating whether the language is preferred, secondary, o
 #lipreads			"Uses lipreading (speechreading)"
 #aac				"Uses augmentative and alternative communication"
 
+ValueSet:       DetectedNotDetectedVS 
+Description:    "Value set consisting of the answers Detected (Present) and NotDetected (Absent). The terms detected and not detected are preferred because they reflect outcomes of experimental observation, while present and absent suggest ground truth. This value set reconciles or combines LOINC answer lists LL956-4"
+SCT#260373001   "Detected (qualifier value)"
+SCT#260415000   "Not detected (qualifier value)"
+SCT#42425007	"Equivocal (qualifier value)"
+SCT#82334004    "Indeterminate (qualifier value)"
+
+ValueSet:		PositiveNegativeVS
```

Comment:
I understand the hesitation to "unknown" however it is a recognized term sometimes included in value sets (e.g.: CGWG FHIR IG profile obs-region-studied).  Can we document in the IG how it should be handled?  e.g.: in _dataAbsentReason_ rather than as an explicit term?